Sources/AWSLambdaRuntimeCore/NewLambda.swift

```diff
@@ -44,4 +45,11 @@ extension Lambda {
             }
         }
     }
+
+    /// The default EventLoop the Lambda is scheduled on.
+    package static var defaultEventLoop: any EventLoop {
+        get {
+            NIOSingletons.posixEventLoopGroup.next()
+        }
+    }
```

Comment:
We must ensure that the eventLoop remains the same, in multiple invocations.

```suggestion

    /// The default EventLoop the Lambda is scheduled on.
    package static let defaultEventLoop: any EventLoop = NIOSingletons.posixEventLoopGroup.next()
```

Add test:
```swift
    @Test
    func testDefaultEventLoopRemainsTheSame() {
        let eventLoop = Lambda.defaultEventLoop
        #expect(eventLoop === Lambda.defaultEventLoop)
    }
```